src/audero-sticky.js

```diff
@@ -432,9 +432,19 @@
     * @return {boolean}
     */
    Sticky.isFeatureSupported = function() {
+      var prefixes = [
+        'moz',
```

Comment:
Hi @ThiRaBrTNK.

Thank you for the contribution. I know this is the standard way of checking for support for such feature (used by other similar libraries) but since it's only a matter of Safari (thus `-webkit`), I'd only add it.